Formula/infracost.rb

```diff
@@ -0,0 +1,78 @@
+class Infracost < Formula
+  desc "Cost estimates for Terraform"
+  homepage "https://docs.infracost.io/"
```

Comment:
Just FYI that we moved docs to https://www.infracost.io/docs/ so we could add /blog in the future too.

Reply:
Sounds good!